server/chat-plugins/chat-monitor.js

```diff
@@ -307,7 +307,7 @@ let statusfilter = function (status, user) {
 	lcStatus = lcStatus.replace('herapist', '').replace('grape', '').replace('scrape', '');
 	// Check for blatant staff impersonation attempts. Ideally this could be completely generated from Config.grouplist
 	// for better support for side servers, but not all ranks are staff ranks or should necessarily be filted.
-	if (/(global|room|upper|senior)?\s*(staff|admin|administrator|leader|owner|founder|mod|moderator|driver|voice|operator|sysop|creator)/gi.test(lcStatus)) {
+	if (/(?:\b|^)(?:staff|admin|administrator|leader|owner|founder|mod|moderator|driver|voice|operator|sysop|creator)(?:\b|$)/gi.test(lcStatus)) {
```

Comment:
is `(?:\b|^)` different from `\b`?

Reply:
```suggestion
	if (/\b(?:staff|admin|administrator|leader|owner|founder|mod|moderator|driver|voice|operator|sysop|creator)\b/gi.test(lcStatus)) {
```

I'm _pretty_ sure this is the exact same thing.

Reply:
It is.